test/src/modules/transformer/apex/__tests__/apex.test.js

```diff
@@ -4,12 +4,30 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { ApexMethod, refreshApex, getSObjectValue } from '../apex';
+import { ApexMethod, BarMethod, refreshApex, getSObjectValue } from '../apex';
+import { ApexMethod as NotSameApexMethod } from '../otherApexConsumer';
+import FooMethod from '@salesforce/apex/FooClass.FooMethod';
```

Comment:
To make this test easier to understand could your rename the `ApexMethod` export to with the actual method name?

```suggestion
import { FooMethod, BarMethod, refreshApex, getSObjectValue } from '../apex';
import { BarMethod as BarMethodFromOther } from '../otherApexConsumer';
import FooMethodDirect from '@salesforce/apex/FooClass.FooMethod';
```

Or even better using namespace import to make it easier to trace each object to the appropriate module.

```suggestion
import * as apex from '../apex';
import * as otheApex from '../otherApexConsumer';
import FooMethod from '@salesforce/apex/FooClass.FooMethod';
```